test/project_types/extension/commands/register_test.rb

```diff
@@ -8,15 +8,13 @@ class RegisterTest < MiniTest::Test
       include TestHelpers::FakeUI
       include ExtensionTestHelpers::TempProjectSetup
       include ExtensionTestHelpers::Messages
-      include ExtensionTestHelpers::Stubs::GetApp
+      # include ExtensionTestHelpers::Stubs::GetApp
```

Comment:
do we need this commented out line?

Reply:
Good catch! This was an oversight on my part.